src/popup/i18n/locales/ja.json5

```diff
@@ -43,6 +43,22 @@
           "zh-CN": "简体中文"
         }
       },
+      // Prompt users to switch the appearance
+      appearance: {
+        // Label for appearance
+        label: "TODO:Appearance",
+        // Description for appearance
+        description: "TODO:Change the appearance to your preferred theme. Changes are only applied to the current browser",
+        // Supported appearances
+        appearances: {
+          // The theme follows the current OS setting
+          device: "TODO:Follow device",
+          // Default light theme
+          light: "TODO:Default light",
+          // A theme which comes with a contrast not so strong as black-on-white and which best fits to night use
+          dark: "TODO:Dimmed dark"
```

Comment:
This pull-request needs translations in Japanese for these strings marked with `TODO:`.
Please make a comment, propose a suggestion here or make another pull-request to this branch `appearance` if you are interested.

Reply:
```
       // Label for appearance
        label: "外観",
        // Description for appearance
        description: "外観を好きなテーマに変更します。変更は現在のブラウザにのみ適用されます。",
        // Supported appearances
        appearances: {
          // The theme follows the current OS setting
          device: "端末の設定に従う。",
          // Default light theme
          light: "ライト（既定）",
          // A theme which comes with a contrast not so strong as black-on-white and which best fits to night use
          dark: "ダーク"
```

Referenced by Firefox settings.

Reply:
Thank you!! I will adopt it!